runtime/near-vm-runner/Cargo.toml

```diff
@@ -14,8 +14,9 @@ This crate implements the specification of the interface that Near blockchain ex
 
 [dependencies]
 cached = "0.12"
-wasmer-runtime = { version = "=0.17.1", features = ["default-backend-singlepass"], default-features = false }
-wasmer-runtime-core = { version = "=0.17.1" }
+wasmer = { version = "=1.0.0-alpha01.0", features = ["singlepass"], default-features = false }
```

Comment:
Does it mean we no longer use singlepass by default with this patch?

Reply:
No, default-features = a set of default features wasmer would use. If set to true or omit it (cargo by default set it to true), wasmer would use the union set of ["singlepass"] and default-features. We don't want that set of features but only "singlepass" feature, just like we don't want them in 0.17.1.